packages/portal-contracts/contracts/messaging/gateway/FuelERC20Gateway.sol

```diff
@@ -92,109 +94,120 @@ contract FuelERC20Gateway is
 
     /// @notice Gets the amount of tokens deposited to a corresponding token on Fuel
     /// @param tokenAddress ERC-20 token address
-    /// @param fuelTokenId ID of the corresponding token on Fuel
+    /// @param fuelContractId ID of the corresponding token on Fuel
     /// @return amount of tokens deposited
-    function tokensDeposited(address tokenAddress, bytes32 fuelTokenId) public view returns (uint256) {
-        return _deposits[tokenAddress][fuelTokenId];
+    function tokensDeposited(address tokenAddress, bytes32 fuelContractId) public view returns (uint256) {
+        return _deposits[tokenAddress][fuelContractId];
     }
 
     /// @notice Deposits the given tokens to an account on Fuel
     /// @param to Fuel account to deposit tokens to
-    /// @param tokenId ID of the token being transferred to Fuel
-    /// @param fuelTokenId ID of the token on Fuel that represent the deposited tokens
+    /// @param tokenAddress Address of the token being transferred to Fuel
+    /// @param fuelContractId ID of the contract on Fuel that manages the deposited tokens
     /// @param amount Amount of tokens to deposit
     /// @dev Made payable to reduce gas costs
-    function deposit(bytes32 to, address tokenId, bytes32 fuelTokenId, uint256 amount) external payable whenNotPaused {
+    function deposit(
+        bytes32 to,
+        address tokenAddress,
+        bytes32 fuelContractId,
+        uint256 amount
+    ) external payable whenNotPaused {
         bytes memory messageData = abi.encodePacked(
-            fuelTokenId,
-            bytes32(uint256(uint160(tokenId))),
-            bytes32(uint256(uint160(msg.sender))), //from
-            to,
-            bytes32(amount)
+            fuelContractId,
+            bytes32(uint256(uint160(tokenAddress))), // OFFSET_TOKEN_ADDRESS = 32
+            bytes32(0), // OFFSET_TOKEN_ID = 64
+            bytes32(uint256(uint160(msg.sender))), //from, OFFSET_FROM = 96
+            to, // OFFSET_TO = 128
+            amount // OFFSET_AMOUNT = 160
```

Comment:
This is no longer wrapped in `bytes`, is there a reason?

Reply:
`abi.encodePacked` just concatenates the values in its bytes representation. An uint256 is equivalent to bytes32 in that regard, so wrapping it is redundant.